update util-linux to 2.27

Cherry pick the update to util-linux 2.27.
We do this to have a new enough sfdisk to work with gpt and fix LP: #1605589.
The version of 2.26.2 is not enough due to sfdisk losing uuids
of gpt partitions on recreate (LP: #1474090).

Upstream buildroot information:
  commit: 4335dbebe5925850f8c3879cfaa77bb6f1dde9e4
  https://git.buildroot.net/buildroot/commit/package/util-linux/util-linux.hash?id=4335dbebe5925850f8c3879cfaa77bb6f1dde9e4
Index: buildroot/package/util-linux/0002-program-invocation-short-name.patch
===================================================================
--- buildroot.orig/package/util-linux/0002-program-invocation-short-name.patch	2015-05-31 21:15:34.000000000 +0000
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,11 +0,0 @@
---- a/configure.ac	2013-09-05 15:31:21.590981268 -0500
-+++ b/configure.ac	2013-09-05 15:31:29.160981049 -0500
-@@ -387,7 +387,7 @@
- 
- AC_MSG_CHECKING([whether program_invocation_short_name is defined])
- AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
--	#include <argp.h>
-+	#include <errno.h>
- ]], [[
- 	program_invocation_short_name = "test";
- ]])], [
Index: buildroot/package/util-linux/util-linux.hash
===================================================================
--- buildroot.orig/package/util-linux/util-linux.hash	2015-05-31 21:15:34.000000000 +0000
+++ buildroot/package/util-linux/util-linux.hash	2016-07-22 15:02:00.403012264 +0000
@@ -1,2 +1,2 @@
-# From https://www.kernel.org/pub/linux/utils/util-linux/v2.26/sha256sums.asc
-sha256	0e29bda142528a48a0a953c39ff63093651a4809042e1790fbd6aa8663fd9666	util-linux-2.26.2.tar.xz
+# From https://www.kernel.org/pub/linux/utils/util-linux/v2.27/sha256sums.asc
+sha256 fcbf04e3ef8377fc8369af7f9afee341c5eef9b02dd5a7ce71532d0957e86dc7  util-linux-2.27.tar.xz
Index: buildroot/package/util-linux/util-linux.mk
===================================================================
--- buildroot.orig/package/util-linux/util-linux.mk	2015-05-31 21:15:34.000000000 +0000
+++ buildroot/package/util-linux/util-linux.mk	2016-07-22 15:02:25.579012264 +0000
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-UTIL_LINUX_VERSION = $(UTIL_LINUX_VERSION_MAJOR).2
-UTIL_LINUX_VERSION_MAJOR = 2.26
+UTIL_LINUX_VERSION = $(UTIL_LINUX_VERSION_MAJOR)
+UTIL_LINUX_VERSION_MAJOR = 2.27
 UTIL_LINUX_SOURCE = util-linux-$(UTIL_LINUX_VERSION).tar.xz
 UTIL_LINUX_SITE = $(BR2_KERNEL_MIRROR)/linux/utils/util-linux/v$(UTIL_LINUX_VERSION_MAJOR)
 
